src/organization/organization.type.js

```diff
@@ -1,5 +0,0 @@
-import { createActionSet } from 'utils';
-
-export const GET_ORG = createActionSet('GET_ORG');
```

Comment:
Nice this is perfect, using `redux-actions` looks like we don't need a helper utility like `createActionSet` whatsoever